qemu/tests/timerdevice_boot.py

```diff
@@ -169,6 +182,15 @@ def update_clksrc(session, clksrc):
     if params["os_type"] == "linux":
         verify_timedrift(session, is_hardware=True)
 
+    repeat_nums = int(params["repeat_nums"])
+    if params.get_numeric("repeat_nums"):
```

Comment:
Already get the value of `repeat_nums` in L185

Reply:
Updated.